### PR TITLE
Prepare 24.11 release notes

### PR DESCRIPTION
**Pull request by Hoverbear**
This shouldn't be merged until https://github.com/ferrocene/ferrocene/pull/952 is.

### CODE REVIEW

**Comment by Hoverbear** [comment]
This should get backported to 24.11/1.81 (of course)

File: ferrocene/doc/release-notes/src/24.11.0.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
```
Comment (pietroalbini) [inline ferrocene/doc/release-notes/src/24.11.0.rst:3]: On the main branch we should mark it as upcoming.

File: ferrocene/doc/release-notes/src/24.11.0.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Ferrocene 24.11.0
+======================
+
+Ferrocene 24.11.0 is the fourth major release of Ferrocene.
```
Comment (Hoverbear) [inline ferrocene/doc/release-notes/src/24.11.0.rst:7]: Add medical

File: ferrocene/doc/release-notes/src/24.11.0.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+qualification of the ``rustfmt`` code formatter, and qualification of the
+:target:`aarch64-unknown-nto-qnx710` and
+:target:`x86_64-pc-nto-qnx710` targets.
+
+New features
+------------
+
+* Ferrocene is now supported and qualified for use in medical devices (IEC 62304).
+* The ``rustfmt`` code formatter version |rustfmt_version| is now supported and
+  qualified for safety critical use.
```
Comment (Urhengulas) [inline ferrocene/doc/release-notes/src/24.11.0.rst:21]:
```suggestion
* The ``rustfmt`` code formatter is now supported and qualified for safety
  qualified for safety critical use.
```

**Review by Urhengulas — COMMENTED** [review]
I don't think we need to specify the version. Also if it is specified like this it would always update which I don't think makes sense in release notes.

File: ferrocene/doc/release-notes/src/24.11.0.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+qualification of the ``rustfmt`` code formatter, and qualification of the
+:target:`aarch64-unknown-nto-qnx710` and
+:target:`x86_64-pc-nto-qnx710` targets.
+
+New features
+------------
+
+* Ferrocene is now supported and qualified for use in medical devices (IEC 62304).
+* The ``rustfmt`` code formatter is now supported and qualified for safety
+  qualified for safety critical use.
```
Comment (pietroalbini) [inline ferrocene/doc/release-notes/src/24.11.0.rst:21]: Duplicated "qualified for safety"?

Comment (Urhengulas) [reply ferrocene/doc/release-notes/src/24.11.0.rst:21]: Sorry, my bad

```suggestion
* The ``rustfmt`` code formatter is now supported and qualified for safety
  critical use.
```

**Review by Urhengulas — APPROVED** [review]

**Comment by Urhengulas** [comment]
bors merge

**Pull request merged**